api/src/beacons/beacon.schema.ts

```diff
@@ -3,6 +3,13 @@ import { HydratedDocument } from 'mongoose';
 
 export type BeaconDocument = HydratedDocument<Beacon>;
 
+export enum Tags {
+  Labour = 'Labour',
+  Tools = 'Tools',
+  Tech = 'Tech',
+  Social = 'Social',
```

Comment:
Very small thing, but I think convention is to have the keys be all uppercase and the values be all lowercase.